.github/workflows/black_linter.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         python -m pip install --upgrade pip
         pip install black==19.3b pytest
         pip install click==8.0.2
-        pip install -r app/requirements/mysql.txt
+        pip install -r app/requirements/postgres.txt
```

Comment:
I add done this with mysql because for testing purposes I could not find a way to generate automatically the postgres DB, whereas it was easy with mysql

Reply:
Yes, but with #183 this fails now, because dependencies are not met. Do you need help to fix this? Shall I look into this or do you want to fix it?

Reply:
Side note, I think because of #194 sqlite is used anyways.

Reply:
I will try to fix this in #197

Reply:
@Bachibouzouk overtakes #197 now